### PR TITLE
TCM-136 | Build dummy cards screen

### PR DESCRIPTION
**Pull request by halilyesilyurt**
### Summary

- [x] Completed dummy cards screen.
- [x] Added navigation to test utility.

### Issue

[TCM-136](https://hyfnla.atlassian.net/browse/TCM-136)

### Video/Screenshot

| Has Card | No Card|
|--------|------|
|cards_has_card|cards_no_card|

### CODE REVIEW

**Review by onodude — DISMISSED** [review]
thanks!

File: app/src/main/java/com/trx/consumer/models/common/CardModel.kt [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    var number: String = "",
+    var creditCardType: String = "",
+    var type: String = "",
+    var securityCode: String = "",
+    var expMonth: String = "",
+    var expYear: String = "",
+    var zip: String = ""
+) : Parcelable {
+
+    val typeText: Int
```
Comment (VivekV95) [inline app/src/main/java/com/trx/consumer/models/common/CardModel.kt:20]: Let's give this @get:StringRes

File: app/src/main/java/com/trx/consumer/models/common/CardModel.kt [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                "amex" -> {
+                    R.string.cards_type_text_amex
+                }
+                else -> {
+                    R.string.cards_type_text_credit_card
+                }
+            }
+        }
+
+    val imageName: Int
```
Comment (VivekV95) [inline app/src/main/java/com/trx/consumer/models/common/CardModel.kt:38]: Let's give this @get:DrawableRes

File: app/src/main/java/com/trx/consumer/models/common/CardModel.kt [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    var securityCode: String = "",
+    var expMonth: String = "",
+    var expYear: String = "",
+    var zip: String = ""
+) : Parcelable {
+
+    val typeText: Int
+        get() {
+            return when (creditCardType.lowerCased()) {
+                "mc" -> {
```
Comment (VivekV95) [inline app/src/main/java/com/trx/consumer/models/common/CardModel.kt:23]: Can just be `"mc" -> R.string.cards_type_text_master_card` along with all the other as everything fits in one line

File: app/src/main/java/com/trx/consumer/models/common/CardModel.kt (hunk 1 above)
Comment (VivekV95) [reply app/src/main/java/com/trx/consumer/models/common/CardModel.kt:20]: Ignore the above, we'll make this in an improvement ticket for both trx and xpass

File: app/src/main/java/com/trx/consumer/models/common/CardModel.kt (hunk 3 above)
Comment (VivekV95) [reply app/src/main/java/com/trx/consumer/models/common/CardModel.kt:23]: Ignore the above, we'll make this in an improvement ticket for both trx and xpass

File: app/src/main/java/com/trx/consumer/models/common/CardModel.kt (hunk 2 above)
Comment (VivekV95) [reply app/src/main/java/com/trx/consumer/models/common/CardModel.kt:38]: Ignore the above, we'll make this in an improvement ticket for both trx and xpass

**Review by VivekV95 — APPROVED** [review]

**Review by VivekV95 — CHANGES_REQUESTED** [review]
Conflicts with `develop`

**Review by VivekV95 — APPROVED** [review]

**Review by onodude — APPROVED** [review]

**Pull request merged**